project-plan.md

```diff
@@ -24,50 +24,95 @@ for programming because they facilitate exploratory programming and debugging.
 Common examples include command-line shells such as Bash and Python's REPL.
 
 The deliverable for this project, then, is to create such a REPL generator for
-the Spoofax Language Workbench. This REPL should provide the following features:
-
-* TODO: based on the research report and the minutes of meeting 2016-04-19,
-  make our own list of features that we will propose to Eelco and Gabriel.
-
-Non-features (e.g. features that should *not* be implemented) are:
-
-* TODO: see above.
+the Spoofax Language Workbench.
+
+# The final product
+The final product will meet the following requirements and demands,
+as specified following the MoSCoW method:
+
+* Must-have
+	* Interactive REPL
```

Comment:
The whole goal is to make an interactive REPL, so I think we can remove this bullet and lift all the subbullets a level higher.